### PR TITLE
Flake8 consistency

### PR DESCRIPTION
**Pull request by sylvestre**
If you are ok with these kind of changes, I will fix the rest and make sure they don't come back.

### CODE REVIEW

File: codechecker_lib/host_check.py [hunk 1]
```diff
@@ -40,7 +40,6 @@ def check_zlib():
 # -----------------------------------------------------------------------------
 def check_psycopg2():
     try:
-        import psycopg2
```
Comment (igalex) [inline codechecker_lib/host_check.py]: This line is required. Please add it back.

**Comment by igalex** [comment]
Please fix the issue above. Otherwise it's OK. Thank you again :-)

**Comment by sylvestre** [comment]
@igalex any issue? Thanks

**Comment by igalex** [comment]
Merged with a modification in codechecker_lib/host_check.py. Thanks for the patch :-)

**Pull request merged**